mantidproject/mantid
# Fit only TOF in EngDiffUI

Merge after https://github.com/mantidproject/mantid/pull/29929, will remove Richard's commits once in master
**Description of work.**

This PR removes the ability to fit in dSpacing in the engineering diffraction GUI, and creates table(s) for TOF peak centres converted to dSpacing as part of the output of a fit.

**To test:**
1. Interfaces -> Diffraction -> Engineering Diffraction
2. Load a focused run file in BOTH TOF and dSpacing e.g.
[ENGINX_305500_bank_1.zip](https://github.com/mantidproject/mantid/files/5554626/ENGINX_305500_bank_1.zip)
3. Select the checkboxes to plot both
4. Ensure in the fit property browser that only the TOF ws can be selected
5. Unplot the dSpacing plot
6. Add some peaks of one type to the plot and fit
7. Ensure that these peaks are shown in dSpasing in a ws2d, e.g. `Gaussian_PeakCentre_dSpacing`
8. Try another fit with peaks of different type and a background function or two thrown in
9. Ensure that the tables are made with the correct names

Fixes #28929. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
